src/two/index.html

```diff
@@ -72,22 +71,22 @@
+                    <a href="https://www.packtpub.com/openlayers-create-gis-web-applications-cookbook/book"><img src="images/packt-ol-cookbook.png"></a>
+                    <a href="https://www.packtpub.com/openlayers-create-gis-web-applications-cookbook/book">OpenLayers Cookbook</a><div class="author">Antonio Santiago Perez</div>Available in Print and eBook formats from <a href="https://www.packtpub.com/openlayers-create-gis-web-applications-cookbook/book">Packt Publishing</a>.
                 </li>
                 <li>
-                    <a href="http://www.packtpub.com/openlayers-2-1-javascript-web-mapping-library-beginners-guide/book"><img src="images/packt-ol.jpg"></a>
-                    <a href="http://www.packtpub.com/openlayers-2-1-javascript-web-mapping-library-beginners-guide/book">OpenLayers 2.10 Beginner's Guide</a><div class="author">Erik Hazzard</div>Available in Print and eBook formats from <a href="https://www.packtpub.com/openlayers-2-1-javascript-web-mapping-library-beginners-guide/book">Packt Publishing</a>.
+                    <a href="https://www.packtpub.com/openlayers-2-1-javascript-web-mapping-library-beginners-guide/book"><img src="images/packt-ol.jpg"></a>
+                    <a href="https://www.packtpub.com/openlayers-2-1-javascript-web-mapping-library-beginners-guide/book">OpenLayers 2.10 Beginner's Guide</a><div class="author">Erik Hazzard</div>Available in Print and eBook formats from <a href="https://www.packtpub.com/openlayers-2-1-javascript-web-mapping-library-beginners-guide/book">Packt Publishing</a>.
                 </li>
                 <li>
-                    <a href="http://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5"><img src="images/openlayers_de_1ed.png"></a>
-                    <a href="http://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">OpenLayers (in German)</a><div class="author">Till Adams &amp; Marc Jansen</div>Available from <a href="http://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">Open Source Press</a>.
+                    <a href="https://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5"><img src="images/openlayers_de_1ed.png"></a>
+                    <a href="https://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">OpenLayers (in German)</a><div class="author">Till Adams &amp; Marc Jansen</div>Available from <a href="https://www.opensourcepress.de/de/produkte/OpenLayers/159/978-3-937514-92-5">Open Source Press</a>.
                 </li>
             </ul>
         </div>
         <div class="cleared spaced">
-            <a href="http://www.osgeo.org/" class="centered"><img src="images/osgeo_sm.png"></a>
-            <p>OpenLayers is a project of the <a href="http://www.osgeo.org/">Open Source Geospatial Foundation</a>.</p>
+            <a href="https://www.osgeo.org/" class="centered"><img src="images/osgeo_sm.png"></a>
```

Comment:
Not a problem, but funny that https://www.osgeo.org/ returns a 302 with http://www.osgeo.org/home as the location (not https).